### PR TITLE
RPM

### LINKED ISSUE #240: Create and Distribute LinchPin RPM

### PR DESCRIPTION
**Pull request by abraverm**
Build Linchpin RPM using [rpmvenv](https://github.com/kevinconway/rpmvenv): `rpmvenv linchpin.json --core_version=1.7.4.1`
Installs linchpin to /opt/linchpin with most python libraries (troublesome libraries are system requirement such as python2-libvirt). The user call linchpin just as it was normally installed.
The RPM created symbolic link from /usr/bin/linchpin to /opt/linchpin/bin/linchpin . The user
doesn't have to activate the virtualenv to make Linchpin work. The RPM was tested in Fedora:30 container.

Fixes #240

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ configparser>=3.5.0
 openshift>=0.8.6
 pyasn1>=0.1.9
 gitpython>=2.1.11
+urllib3<1.25
```
Comment (abraverm) [inline requirements.txt]: openshift required this, I'm not sure why pip didn't catch this...

**Comment by abraverm** [comment]
We need to add this to CI and release automation, I still don't have permissions to access it.

**Comment by abraverm** [comment]
Verified with Fedora 30 container:
https://copr.fedorainfracloud.org/coprs/alexbmasis/linchpin/repo/fedora-30/alexbmasis-linchpin-fedora-30.repo

**Comment by abraverm** [comment]
@14rcole  we need to setup CentOS-PaaS-SIG or other account in copr, create repository for linchpin and setup custom source script (the rpmbuild_copr.sh):
```bash
copr edit-package-custom linchpin --name linchpin --script rpmbuild.sh --script-builddeps 'git python-virtualenv python-pip' --script-resultdir results
```

**Comment by 14rcole** [comment]
Are we releasing in copr?  I thought we were just adding the rpm to the release on github.  Althought it does make it easier to install if we push it to copr

**Comment by abraverm** [comment]
something will have to build those RPMs, and copr sounded like a good candidate for this, also it would simplify the installation and maintenance.. I think

**Comment by 14rcole** [comment]
@abraverm sounds good to me.  Can we merge this PR then set up copr?  Or should the accounts be set up first?

**Comment by abraverm** [comment]
I think we can merge it

**Comment by 14rcole** [comment]
[merge]

**Pull request merged**